### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by irma-kurnia-alaska**
# Alaska Airlines Pull Request

**Fixes:** #49 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```
Comment (blackfalcon) [inline demo/index.html:26]: Please update this area to be like the following code example

https://github.com/AlaskaAirlines/auro-avatar/blob/2a95c410cb8e02f91c1c4868305792fc4c1754b1/demo/index.html#L12-L17

File: package.json [hunk 2]
```diff
@@ -25,12 +25,14 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.15.5",
-    "@aurodesignsystem/webcorestylesheets": "^4.8.0"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.2"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.15.5",
```
Comment (blackfalcon) [inline package.json:32]: Remove this dependency

File: src/style.scss [hunk 3]
```diff
@@ -5,12 +5,15 @@
 
 /* stylelint-disable declaration-empty-line-before, max-nesting-depth, selector-max-class, selector-class-pattern, selector-max-combinators, selector-max-compound-selectors, no-descending-specificity, scss/double-slash-comment-empty-line-before, order/properties-order */
 
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
Comment (blackfalcon) [inline src/style.scss:9]:
```suggestion
```

Delete this line, references to CSS variables is not necessary

File: src/style.scss [hunk 4]
```diff
@@ -5,12 +5,15 @@
...
 
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/animation';
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
Comment (blackfalcon) [inline src/style.scss:12]:
```suggestion
```

Remove this line. This is repeated from line 8.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please be sure to update all npm dependencies to latest versions. Also update the repo to support Node 18 || 20 as described in this support policy

https://auro.alaskaair.com/node-support#gsc.tab=0

File: demo/index.html (hunk 1 above)
Comment (irma-kurnia-alaska) [reply demo/index.html:26]: Ok, sure i will updated. Thanks

**Pull request merged**